libs/ui/src/lib/components/button/button.tsx

```diff
@@ -23,7 +23,8 @@ export function Button({ children, button, type, disabled, onClick }: ButtonProp
           </span>
         </>
       )}
-      {button === 'secondary' && children}
+
+      {button !== 'secondary' && children}
```

Comment:
this line is causing an error. 
![image](https://github.com/rp-js/rpjs-website/assets/74775672/425e866f-4703-48ca-990d-2b199e10d74b)
how to fix:
```ts 
 {button === 'secondary' && children}
```